### ISSUE TITLE
Resetting animation on static entity

### ISSUE THREAD

**Issue by benbarefield** [craftyjs/Crafty#414]
If an entity is not moving it seems that calling .reset() doesn't work. The sprite doesn't update back to its first frame. For example:

``` javascript
var resetAnimation = function () { this.reset(); }
Crafty.e("2D, DOM, LeftCannonSprite, SpriteAnimation")
    .attr({x : 200, y : 200 })
    .animate("Shoot", 0, .5, 9)
    .animate("Shoot", 36, 0)
    .bind("AnimationEnd", resetAnimation);
```

Results in the sprite being stuck on the last frame of the animation. If you change resetAnimation to:

``` javascript
resetAnimation = function() {
    this.x += 1;
    this.x -= 1;
    this.reset();
}
```

The sprite properly resets itself.

**Comment by starwed** [comment]
I believe this will be fixed by #426. (grozens changes to SpriteAnimation).   The "Change" event needs to be triggered somehow; modifying this.x was doing this as a side effect.

**Comment by starwed** [comment]
Should be fixed by #426
